MIGRATION.md

```diff
@@ -19,6 +48,55 @@ From version 3.0.0 onwards, you can access documentation for a specific version
 
 You can upgrade to a major version using `npm update contentful`
 
+## Migration to contentful.js 10.x
+Version `10.0.0` is a complete rewrite in Typescript. This version introduces a new concept of [chained clients](#chained-clients). 
+
+### Version compatibility
+- Node: >= 12 (LTS)
+- Chrome: >= 80,
+- Edge >= 80,
+- Firefox >= 75,
+- Safari >= 13,
+
+We completely dropped support for old IE browsers (no `legacy` bundle)
+> You can always find the supported browsers in our shared [browserlist-config](https://github.com/contentful/browserslist-config/blob/master/index.js)
+
+### Client config
+The fields `resolveLinks` and `removeUnresolved` have been removed from the TS config interface for `createClient`. 
+We still respect them for now, but you will see a warning in the console when using them. Instead, you should use one of the [chained clients](./README.md#chained-clients) to achieve the same result.
```

Comment:
This text should be updated (we no longer support them), and I would also mention that it's a breaking change (maybe a "Breaking changes" section?)